src/Adapter/Customer/QueryHandler/GetCustomerForViewingHandler.php

```diff
@@ -273,7 +289,7 @@ private function getCustomerOrders(Customer $customer)
         }
 
         return new OrdersInformation(
-            Tools::displayPrice($totalSpent),
+            $this->locale->formatPrice($totalSpent, $this->context->getContext()->currency->iso_code),
```

Comment:
I'm wondering if we shouldn't use the order currency I know it changes the initial behaviour, but maybe it was forgotten?

Reply:
I don't think it's a good idea. It is completely out of the scope of this PR.

Reply:
May be you could create a dedicated issue ?

Reply:
I agree with @Progi1984 , if the QA team finds an issue related to this the whole PR could be blocked for a while...

By the way are there cases when the order currency is different than the context currency ?

Reply:
Yes, I know it's not in this PR scope But it makes me wonder it would be worth investigating on the expected result here
And yes I think in some cases the currency can be different between the order and the context, especially in an Order history If it's the order you are purchasing then it should be the same logically